scripts/fuzz_shell.js

```diff
@@ -172,6 +195,48 @@ var imports = {
     'table-set': (index, value) => {
       exports.table.set(toAddressType(exports.table, index), value);
     },
+
+    // Export operations.
+    'call-export': (index) => {
+      callFunc(getExportByIndex(index));
+    },
+    'call-export-catch': (index) => {
+      try {
+        callFunc(getExportByIndex(index));
+        return 0;
+      } catch (e) {
+        // We only want to catch exceptions, not wasm traps: traps should still
+        // halt execution. Handling this requires different code in wasm2js, so
+        // check for that first (wasm2js does not define RuntimeError, so use
+        // that for the check).
+        var wasm2js = !WebAssembly.RuntimeError;
```

Comment:
This seems to assume we only fuzz wasm2js in environments that don't support WebAssembly. Otherwise `WebAssembly.RuntimeError` would be defined even if we were using wasm2js, right? I don't understand why this does what we want.

Reply:
Ah, no, when we run wasm2js it does `WebAssembly = { .. polyfill .. }`, so it tramples any native support. We can't mix native and polyfill stuff, so we have to do that.

Reply:
Got it, thanks. It would be good to clarify that in the comment.